examples/protocol_specification_ex/ledger_api.yaml

```diff
@@ -0,0 +1,37 @@
+---
+name: ledger_api
+author: fetchai
+version: 0.1.0
+description: A protocol for ledger APIs requests and responses.
+license: Apache-2.0
+aea_version: '>=0.4.0, <0.5.0'
+speech_acts:
+  get_balance:
+    ledger_id: pt:str
+    address: pt:str
+  transfer:
```

Comment:
Note, as mentioned on Slack this won't be possible as ledger apis don't have access to keys; so a signed tx must be sent instead.